### PR TITLE
categories loaded in

### PR DESCRIPTION
**Pull request by Hm255**

### CODE REVIEW

File: frontend/nc-games/src/App.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ function App() {
             <Route path="/" element={<Welcome />} />
             <Route path="/reviewlist" element={<Reviewlist />} />
             <Route path="/categories" element={<Categories />} />
+            <Route path="/reviewlist/:category" element={<Reviewlist />}/>
```
Comment (Rosem10) [inline frontend/nc-games/src/App.js]: Your reviewlist component is still named inconsistently (the reviewlist file itself is not capitalised, but the places in which it is being called are). Same thing with navigation - and it means your code won't run when cloned. Please make sure this is consistent across the app!

File: frontend/nc-games/src/components/Categories.jsx [hunk 2]
```diff
@@ -15,7 +20,16 @@ export default function Categories() {
   return (
     <ul>
       {categories.map((category) => {
-        return <Category key={category.category_name} category={category} />;
+        return  <li className="categories" key={category['slug']}> 
+        <p className="categoryName">Category:</p><p className="catName">{category['slug']}</p>
```
Comment (Rosem10) [inline frontend/nc-games/src/components/Categories.jsx:24]: This is nice!

File: frontend/nc-games/src/components/navigation.jsx [hunk 3]
```diff
@@ -11,20 +11,21 @@ export default function Navigation () {
...
-    return (<div>
+    return (
+    <div>
         <h1>Welcome to NC games {name}</h1>
         <Link to="/reviewlist">
         <button onClick={toReviews}>Reviews</button>
         </Link>
-        {/* <Link to="/Categories">
+        <Link to="/Categories">
         <button onClick={toCategories}>Categories</button>
```
Comment (Rosem10) [inline frontend/nc-games/src/components/navigation.jsx]: This would be better named as something more descriptive of what it's doing e.g. 'handleClick'

**Review by Rosem10 — APPROVED** [review]
I've left you a couple of hints, but otherwise - it's looking really good!

**Comment by itskatherine** [comment]
Hi Hamza, similarly to what Rose said above, this code is currently not running on my end as your components aren't named correctly and it's throwing errors.
Can you please change the filename of your component currently named 'reviewList.jsx' to 'Reviewlist.jsx'
And your component named 'navigation.jsx' to 'Navigation.jsx'

Once you've done this can you please resubmit the PR 🙂

File: frontend/nc-games/src/components/reviewList.jsx [hunk 4]
```diff
@@ -1,29 +1,51 @@
 import React, {useState, useEffect} from "react";
```
Comment (Rosem10) [inline frontend/nc-games/src/components/reviewList.jsx:1]: The name of this file and navigation.jsx themselves need to be changed too, please! (in the file navigator)

File: frontend/nc-games/src/components/Review.jsx [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+//     const [review, setReview] = useState([]);
+//     const [loading, setLoading] = useState(true);
+
+//     useEffect(()=>{ 
+//         getReview(id)
+//         .then((data) => {
+//             console.log(data)
+//             if(id === undefined){   
+                
+//                 setReview(<h1>where do you think you're going</h1>);
```
Comment (Rosem10) [inline frontend/nc-games/src/components/Review.jsx:15]: Please do not commit commented code!

**Review by Rosem10 — APPROVED** [review]
Just a couple of **_tiny_** changes to make. You don't need to do another PR, just ensure they're resolved before moving onto the next ticket!

**Pull request merged**